improver_tests/wxcode/wxcode/test_WeatherSymbols.py

```diff
@@ -1315,11 +1331,9 @@ def test_error_on_unexpected_comparison(self):
         one of the expected strings."""
         arr = np.array([0, 1, 2], dtype=np.int32)
         plugin = WeatherSymbols(wxtree=wxcode_decision_tree())
-        msg = (
-            "Invalid comparator: !=. ",
-            "Comparator must be one of '<', '>', '<=', '>='.",
-        )
-        with self.assertRaises(ValueError, msg=msg):
+        msg = "Invalid comparator: !=. Comparator must be one of '<', '>', '<=', '>='."
+
+        with self.assertRaisesRegex(ValueError, msg):
```

Comment:
Previously implemented incorrectly. Fixed in this PR, but unrelated to the change being implemented.